apps/etherpad/vars/vault/main.yml.j2

```diff
@@ -0,0 +1,11 @@
+# customer: {{ customer }}
+# env_type: {{ env_type }}
+
+# postgresql
+{% set postgresql_credentials = databases.postgresql | json_query("[?release=='" ~ etherpad_postgresql_version ~ "'].databases | [0][?application=='etherpad'].{user: user, password: password} | [0]") %}
```

Comment:
why not separate the variables to make it simpler?

Reply:
If I you mean separating `user` from `password`, by doing so, you will duplicate the same request with the same complexity. This request is used for almost every app vault file.